packages/core/src/types/converters.ts

```diff
@@ -1412,8 +1416,8 @@ export function axelarTransferInfoFromJSON(
   value: AxelarTransferInfoJSON,
 ): AxelarTransferInfo {
   return {
-    srcChainID: value.src_chain_id,
-    dstChainID: value.dst_chain_id,
+    fromChainID: value.from_chain_id,
+    toChainID: value.to_chain_id,
```

Comment:
Implemented in this way it's a breaking change right?

But in the API we made it additive, why not do the same here?

Reply:
Updated to make everything additive